content/docs/command-reference/remote/modify.md

```diff
@@ -582,6 +582,69 @@ more information.
 
 </details>
 
+<details>
+
+### Click for WebDAV
+
+- `token` - token for WebDAV server, can be empty in case of using
+  `user/password` authentication.
+
+  ```dvc
+  $ dvc remote modify --local myremote token mytoken
```

Comment:
Does it need quotes like other string params? I.e.:
```suggestion
  $ dvc remote modify --local myremote token "mytoken"
```

Reply:
I don't know, thought it to be more like a variant of `user`, which has no quotes? But adding quotes should probably be fine. Are there guidelines what parameters are considered as strings/numerics/... and what formatting to apply to each?

Reply:
I think it's more of a command line parser thing. All strings can be wrapped with `"` but if they have special chars or spaces then you definitely need to wrap them. And in fact `'` may be better... Idk. I'll just commit this!

Reply:
Actually, have you tried this? I assume you tested all the WebDAV params so you can probably tell or double check whether quotes are needed.

Reply:
Or if you can rec a quick WebDAV server software to install and test locally, please share 🙂

Reply:
Ok, I think you are right, it is better to wrap this in `'`. There seems to be no standard defining the content and structure of tokens for WebDAV (at least it should be some kind of string). What I can tell for the service I am using (it is basically an onwcloud): it supports OAuth 2.0, probably bearer tokens, which might already contain some basic punctuation characters.

Reply:
Thanks for the tip. Resolving

Reply:
> Or if you can rec a quick WebDAV server software to install and run locally for testing, please share slightly_smiling_face

Check this link: https://doc.owncloud.com/server/admin_manual/installation/docker/#docker-compose-yaml-file

Copy-paste that into a `docker-compose.yml` file and then `docker-compose up`.